test/testdata/infer/pinning_static_field.rb.autocorrects.exp

```diff
@@ -0,0 +1,23 @@
+# -- test/testdata/infer/pinning_static_field.rb --
+# typed: true
+
+StaticField1 = T.let(0, Integer)
+def example1
+  x = StaticField1
```

Comment:
This "fixes" the change by not suggesting an autocorrect for the pinning error.

I didn't look into how hard it would be to change the location information, but one thing is that right now the TypeAndOrigins information in e.g. "expected/got" errors will show that it got `Integer` because of the type annotation on the static field.

Maybe that's fine to give up? When a variable gets its type from

```ruby
x = foo
```

we don't show the sig for `foo` in the "expected/got" error message.

Anyways, I figure that can be a change for another day if someone wants to look into it. At least this makes it so that `-a` doesn't suggest syntax errors.